test-scripts/jenkins-run-tests-get-results.sh

```diff
@@ -16,7 +16,7 @@ flock -x 200 || exit 1
 
 # To work around memory leak, kill NetComm and restart the teststand
 # (the teststand dies when NetComm is killed)
-SSH_RESTART_NETCOMM="ssh -t ${ROBOT_ADDRESS} sh -c 'killall FRC_NetCommDaemon; sleep 1; /etc/init.d/teststand stop; /etc/init.d/teststand start; sleep 1'"
+SSH_RESTART_NETCOMM="ssh -t ${ROBOT_ADDRESS} sh -c 'killall FRC_NetCommDaemon; sleep 1; /etc/init.d/teststand stop'"
```

Comment:
Why are we stopping the teststand if we aren't using it? Does it get started on roboRIO startup or something?

Reply:
Any old PR's would need it running, and this PR, and once this gets merged anything after it would need it killed. So this only kills it to ensure its not running.

Reply:
Yes, if the package is installed.  Although it should get killed when the NetCommDaemon is killed, I wasn't counting on that before either.